mailqueue/models.py

```diff
@@ -143,7 +143,10 @@ def _send(self):
 class Attachment(models.Model):
     file_attachment = models.FileField(storage=get_storage(), upload_to=upload_to,
                                        blank=True, null=True)
-    original_filename = models.CharField(default=None, max_length=250, blank=False)
+    # Note: for original_filename, null=True is defined only to simplify the migration to the new version of this 
```

Comment:
I had added these verbose comments, so that we'll understand why we did this fix, if we'll dig into this ZPD fork in the future.